VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -85,6 +90,20 @@ public async Task<IActionResult> GetTestCallResultForParticipantAsync(Guid confe
             }
         }
 
+        private async Task<Guid> GetIdForParticipantByUsernameInConference(Guid conferenceId, string username)
+        {
+            var conference = _conferenceCache.GetConference(conferenceId);
+            if (conference == null)
+            {
+                var conferenceDetail = await _videoApiClient.GetConferenceDetailsByIdAsync(conferenceId);
+                await _conferenceCache.AddConferenceToCache(conferenceDetail);
+                conference = _conferenceCache.GetConference(conferenceId);
+            }
+
+            return conference.Participants
```

Comment:
Might throw here... Just in case there are no Participants, which will probably never going to happen, but could be some defensive coding ??

Reply:
If it doesn't exist, it will throw an exception because of the Single() and not SingleOrDefault() method